### PR TITLE
tests: add load unit interruption test

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 
test load unit abnormal interrupt during processing, check whether checkpoint in loader works as expected
ATTENTION: either this pr or #112 is merged first, the other one must fix to use the global dmctl binaries

### What is changed and how it works?
use failpoint to control load unit abnormal interruption.

### Check List 

Tests 

 - Integration test

### CODE REVIEW

File: loader/loader.go [hunk 1]
```diff
@@ -136,6 +137,13 @@ func (w *Worker) run(ctx context.Context, fileJobQueue chan *fileJob, workerWg *
 				offsetSQL := w.checkPoint.GenSQL(job.file, job.offset)
 				sqls = append(sqls, offsetSQL)
 
+				failpoint.Inject("LoadExceedOffsetExit", func(val failpoint.Value) {
+					threshold, _ := val.(int)
+					if job.offset >= int64(threshold) {
+						panic(fmt.Sprintf("load offset %d execeeds threshold %d", job.offset, threshold))
```
Comment (csuzhangxc) [inline loader/loader.go]: should we include any marker (like `[failpoint]`) in the log/message triggered by failpoint? so we can diff them from normal log/message. see a panic message may make us panic until we search it in the codebase.

File: tests/load_interrupt/dmctl.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	"github.com/pingcap/dm/tests/utils"
+)
+
+func main() {
+	cli, err := utils.CreateDmCtl("127.0.0.1:8261")
+	if err != nil {
+		utils.ExitWithError(err)
+	}
+	conf := os.Args[1]
+	err = utils.StartTask(context.Background(), cli, conf, nil)
```
Comment (csuzhangxc) [inline tests/load_interrupt/dmctl.go]: DNM this PR until #112 merged and re-use dmctl binaries?

File: loader/loader.go (hunk 1 above)
Comment (amyangfei) [reply loader/loader.go]: addressed

File: tests/load_interrupt/dmctl.go (hunk 2 above)
Comment (amyangfei) [reply tests/load_interrupt/dmctl.go]: not big problem, either this pr or #112 is merged first, the other one must fix to use the global dmctl binaries

**Comment by amyangfei** [comment]
PTAL @GregoryIan

File: tests/load_interrupt/run.sh [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    check_port_alive $WORKER1_PORT
+    check_port_alive $WORKER2_PORT
+
+    cd $cur && GO111MODULE=on go build -o bin/dmctl && cd -
+    $cur/bin/dmctl "$cur/conf/dm-task.yaml"
+
+    check_port_offline $WORKER1_PORT 20
+    check_port_offline $WORKER2_PORT 20
+
+    run_sql "SELECT count(*) from dm_meta.test_loader_checkpoint where cp_schema = '$TEST_NAME' and offset < $THRESHOLD" $TIDB_PORT
```
Comment (GregoryIan) [inline tests/load_interrupt/run.sh]: can we estimate how many rows are there?

Comment (amyangfei) [reply tests/load_interrupt/run.sh]: added

**Comment by GregoryIan** [comment]
LGTM. well done

**Pull request merged**